src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -896,6 +906,28 @@ public String getEntityNameParamName(SingleResourceNameConfig resourceNameConfig
     return localVarName(Name.from(resourceNameConfig.getEntityName(), "name"));
   }
 
+  /////////////////////////////////////// Page Streaming ////////////////////////////////////////
+
+  /** The formatted field name of a page streaming request token. */
+  public String getRequestTokenFieldName(PageStreamingConfig pageStreaming) {
+    return pageStreaming.getRequestTokenField().getSimpleName();
```

Comment:
The term "field key" is used in other parts - e.g. `getFieldKey`. I would suggest implementing `keyName` in `NodeNameFormatter` correctly for Node and use that instead of returning the bare simple name.